server/src/v1/controllers/meetup.js

```diff
@@ -22,12 +22,10 @@ client.connect((err) => {
  * @param  {object} res
  */
 const createMeetup = (req, res) => {
-    const sqlQueryString = `INSERT INTO meetups (title, location, created_on, topic, images, tags, happening_on)
-                            VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING *`;
+    const sqlQueryString = `INSERT INTO meetups (location, topic, images, tags, happeningon)
```

Comment:
Well done @Amriesgrace Have you tried using `happeningOn`instead?

Reply:
Will do that